02_configure_host.sh

```diff
@@ -88,7 +88,7 @@ ansible-playbook \
     -e "vm_platform=$NODES_PLATFORM" \
     -e "sushy_ignore_boot_device=$REDFISH_EMULATOR_IGNORE_BOOT_DEVICE" \
     -e "manage_baremetal=$MANAGE_BR_BRIDGE" \
-    -e "provisioning_url_host=$PROVISIONING_URL_HOST" \
+    -e "provisioning_url_host=${PROVISIONING_URL_HOST:-}" \
```

Comment:
I think this is only used to template the kernel/ramdisk URL in metal3-dev-env which isn't actually used in dev-scripts:

```
$ grep -Ri PROVISIONING_URL_HOST ./*
./roles/libvirt/defaults/main.yml:provisioning_url_host: 172.22.0.1
./roles/libvirt/templates/ironic_nodes.json.j2:        "deploy_kernel": "http://{{ provisioning_url_host }}/images/ironic-python-agent.kernel",
./roles/libvirt/templates/ironic_nodes.json.j2:        "deploy_ramdisk": "http://{{ provisioning_url_host }}/images/ironic-python-agent.initramfs"
```

However for consistency, I think we should just set this to `PROVISIONING_HOST_IP` which is defined, and would be the correct IP if someone wanted to test via Ironic directly when also defining `IRONIC_IPA_DOWNLOADER_LOCAL_IMAGE` to add the IPA images to the virthost httpd?